extras/free-books.md

```diff
@@ -87,6 +87,9 @@ Name | Author(s)
 ### Cloud Computing
 
 ### Math (Linear Algebra)
+Name | Author(s) 
+:-- | :--: 
+[Linear Algebra](https://www.amazon.com/Linear-Algebra-2nd-Kenneth-Hoffman/dp/0135367972) | Kenneth M. Hoffman
```

Comment:
@arpit1997 The link points to a paid resource, so it's out of scope in this file.